packages/kbn-io-ts-utils/src/to_number_rt/index.ts

```diff
@@ -8,7 +8,7 @@
 
 import * as t from 'io-ts';
 
-export const toNumberRt = new t.Type<number, unknown, unknown>(
+export const toNumberRt = new t.Type<number, number, unknown>(
```

Comment:
Changed this to satisfy SavedObjectClient type. Was there any specific reason to keep it as `unknown` ? Because so far the tests seem to be fine 🤷 
Thanks!

Reply:
I'm not sure, it might be due to our specific usage. Where is the SavedObjectClient type you're using this for?